rfcs/003-workspaces.md

```diff
@@ -101,14 +101,31 @@ Course staff will declare workspace config per question via `workspace` in `info
 ```json
 {
     "workspace": {
-        "image": "some-docker-image"
+        "image": "some-docker-image-name"
     }
 }
 ```
 
-The home directory in the workspace will be determined by the `workspace` directory inside a question directory. In the future, we'll add the ability to dynamically generate files via `server.py` and place them into the home directory. This is not part of the MVP.
+The home directory in the workspace will be determined by the `workspace_homedir` directory inside a question directory. In the future, we'll add the ability to dynamically generate files via `server.py` and place them into the home directory. This is not part of the MVP.
```

Comment:
Why this change? We also don't use files/folders with underscores anywhere else in PL.